model/Core/Classes/ExternalIdentifier.md

```diff
@@ -0,0 +1,31 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# ExternalIdentifier
+
+## Summary
+
+TODO
+
+## Description
+
+TODO
+
+
+## Metadata
+
+- name: ExternalIdentifier
+- SubclassOf: none
+- Instantiability: Concrete
+
+
+## Properties
+
+- externalIdentifierType
+  - type: ExternalIdentifierType
+- identifier
+  - type: xsd:string
```

Comment:
same here; needs `minCount: 1` and `maxCount: 1`

Reply:
Fixed.